tedana/decomposition/__init__.py

```diff
@@ -3,6 +3,6 @@
 
 from .pca import tedpca
 from .ica import tedica
+from .ma_pca import pca
```

Comment:
Super small -- can we not call this just `PCA` ? i'd like to support e.g. 95% variance threshold with sklearn's PCA. What if we just call it `ma_pca` as we call the module ?

Reply:
Agree

Reply:
Done!